worker/uniter/uniter.go

```diff
@@ -452,6 +459,9 @@ func (u *Uniter) loop(unitTag names.UnitTag) (err error) {
 			Actions: actions.NewResolver(
 				u.logger.Child("actions"),
 			),
+			VerifyCharmProfile: verifycharmprofile.NewResolver(
```

Comment:
Probably don't set this on CAAS if it shouldn't be used.

Reply:
See comment here https://github.com/juju/juju/pull/11701/files#r439285849

Checking if it's a CAAS model should be an anti-pattern. See MAAS versions as a prior discussion.